src/autocomplete/autocomplete.ts

```diff
@@ -12,7 +12,7 @@ let nextID: number = 0;
 export class Autocomplete {
   @observable public inputValue: string  = '';
   @bindable public service: any;
-  @bindable({ defaultBindingMode: bindingMode.twoWay }) public value: string;
+  @bindable({ defaultBindingMode: bindingMode.twoWay }) public value: any;
```

Comment:
welke andere vlaue kan er hier nog zijn buiten string?

Reply:
Met de nieuwe adresregister that is een object (IGemeente, IStraat, ..) als dat gevonden wordt, anders is een string.

Reply:
ah check